### DISCUSSION TITLE
how to turn off dark mode on mac?

### DISCUSSION THREAD

**Discussion by tatertotsey** [mortbopet/Ripes#171] [Q&A] [answered]
Hey, 

How can I turn off the dark mode on mac or is there a setting to change editor's text colour?

Thanks in advance!

**Comment by mortbopet** [comment, marked as answer]
Hi @tatertotsey 

I don't have a mac myself so i'm unable to test it, but according to [this port](https://forum.qt.io/topic/99799/qt-5-12-1-and-dark-mode-on-macos-mojave-10-14/8) you could try to add the following to the `Info.plist` file inside the application bundle https://github.com/agarny/opencor/blob/9a9a614a08dc75b9807150ab5fce101150c0b091/distrib/macos/Info.plist.in#L51

**Reply by tatertotsey** [reply]
@mortbopet it worked! Thanks!
